cluster/ubuntu/util.sh

```diff
@@ -196,6 +196,7 @@ EOF
 function create-kube-apiserver-opts() {
   cat <<EOF > ~/kube/default/kube-apiserver
 KUBE_APISERVER_OPTS="\
+ --bind-address=0.0.0.0\
```

Comment:
`--advertise-address=<nil>: The IP address on which to advertise the apiserver to members of the cluster. This address must be reachable by the rest of the cluster. If blank, the --bind-address will be used. If --bind-address is unspecified, the host's default interface will be used.`
fyi.
I use `--bind-address` instead of `--advertise-address` here